README.md

```diff
@@ -1,13 +1,15 @@
 [![Build Status](https://travis-ci.org/jquery/jquery-migrate.svg?branch=master)](https://travis-ci.org/jquery/jquery-migrate)
 [![Coverage Status](https://img.shields.io/coveralls/jquery/jquery-migrate.svg?style=flat)](https://coveralls.io/r/jquery/jquery-migrate?branch=master)
 
-# Migrate older jQuery code to jQuery 3.0+
+#### NOTE: To upgrade to jQurey 3.0 you first need version 1.12.x or 2.2.x. If you're using an older version, first upgrade to one of these versions using [jQuery Migrate 1.x](https://github.com/jquery/jquery-migrate/tree/1.x-stable#readme) to resolve any compatibility issues. Then you can upgrade to 3.0 using jQuery Migrate 3.0. For more information about the changes made in jQuery 3.0, see the [upgrade guide](http://jquery.com/upgrade-guide/3.0/) and [blog post](https://blog.jquery.com/2016/06/09/jquery-3-0-final-released/).
 
-#### NOTE: If you have not yet upgraded to 1.12.x or 2.2.x, first upgrade to them and use [jQuery Migrate 1.x](https://github.com/jquery/jquery-migrate/tree/1.x-stable#readme) to resolve any compatibility issues before using jQuery Migrate 3.0 and upgrading to jQuery 3.0.
+# jQuery Migrate
+Upgrading jQuery can be painful, when breaking changes have been introduced.
```

Comment:
I don't think it's a negative aspect of jQuery that upgrading can be painful. It's in the nature of frameworks and libraries that have their own ecosystems, but you might be right that it can be taken that way so I'll remove this. :)